tools/pneumocat/pneumocat.xml

```diff
@@ -0,0 +1,80 @@
+<tool id="pneumocat" name="PneumoCaT" version="@VERSION@">
+    <description> Pneumococcal Capsular Typing of illumina reads</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+    
+    #import os
+
+    #if $input.type == "paired"
+        #set $for_input = "{}.fastq".format(os.path.splitext($input.forward.name)[0])
+        #set $rev_input = "{}.fastq".format(os.path.splitext($input.reverse.name)[0])
+        ln -s  "$input.forward" ./$for_input &&
+        ln -s  "$input.reverse" ./$rev_input &&
+
+    #else
+        #set $for_input = "{}.fastq".format(os.path.splitext($input.fastq_collection.forward.name)[0])
+        #set $rev_input = "{}.fastq".format(os.path.splitext($input.fastq_collection.reverse.name)[0])
+        ln -s "$input.fastq_collection.forward" $for_input &&
+        ln -s "$input.fastq_collection.reverse" $rev_input &&
+
+    #end if
+
+    PneumoCaT.py -1 "$for_input" -2 "$rev_input" -o outputs --threads "\${GALAXY_SLOTS:-1}" --cleanup
```

Comment:
please use single-quotes, please see here for more details: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#command-formatting